### PR TITLE
ESL Alert extended and became releasable

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: src/modules/esl-alert/README.md [hunk 1]
```diff
@@ -1,34 +1,53 @@
...
+ESLAlert is a component to show small notifications on your pages. ESLAlert can have multiple instances on the page.
 
 ---
 
 ##Usage
 
-First of all, register component using static `ESLAlert.init` method.
-Then the following events can be dispatched globally (on window) to control alert: 
+First, use the common approach to register component: `ESLAlert.register()`.
+Then you can attach `<esl-alert>` component in the expected place in the document or initialize it globally using `ESLAlert.init` shortcut.
```
Comment (dshovchko) [inline src/modules/esl-alert/README.md:14]: Should be 'globally **by** using' instead 'globally using'

File: src/modules/esl-alert/README.md [hunk 2]
```diff
@@ -1,34 +1,53 @@
...
+By default, the target to catch alerts is the `esl-alert` parent element. 
+Target can be changed using `target` attribute with the [TraversingQuery](../esl-traversing-query/README.md) support, 
+or through the `$target` property that accepts any EventTarget instance (including the window). 
+
+ESL Alert listens to the following events: 
 - `esl:alert:show` to show alert
 - `esl:alert:hide` to hide alert
 
-Use custom event details to customize alert.
+Use CustomEvent `details` to customize alert. Alert `details` accept the following properties:
```
Comment (dshovchko) [inline src/modules/esl-alert/README.md:25]: ? accept**s**

**Review by dshovchko — APPROVED** [review]

**Review by Sisha3342 — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Review by andreybelous — APPROVED** [review]

**Pull request merged**